chat-widget/src/plugins/newMessageEventHandler.ts

```diff
@@ -46,10 +46,6 @@ export const createOnNewAdapterActivityHandler = (chatId: string, userId: string
                 if (activity?.channelData?.tags?.includes(Constants.systemMessageTag)) {
                     // eslint-disable-next-line @typescript-eslint/no-explicit-any
                     payload.messageType = Constants.systemMessageTag;
-                    TelemetryHelper.logActionEvent(LogLevel.INFO, {
```

Comment:
Why remove this?

Reply:
Because it was using MessageReceived event, so would confused with agent MessageReceived and system MessageReceived. Anyways, created a new event `SystemMessageReceived`